.github/workflows/continuous-integration.yml

```diff
@@ -1,10 +1,7 @@
 name: CI
 
 on:
-  push:
-    branches: [ main ]
   pull_request:
-    branches: [ main ]
```

Comment:
Won't this disable CI for any pull request against main?

Reply:
No, if there are no filters then it rans for all PRs